e2e/mandelbrot/mandelbrot.py

```diff
@@ -6,6 +6,14 @@
 #import numpy as np
 import torch_np as np
 
+
+# To run on CUDA, change "cpu" to "cuda" below.
+# NB: set_default_device("cpu") leads to fails if cuda.is_available() is False, hence guard it.
```

Comment:
I can't repro that on my laptop
![image](https://user-images.githubusercontent.com/3291265/235436138-6f81fc1e-ce91-4047-aca6-cee170eeac95.png)

If you can repro it, can you report it as a bug?

Reply:
Nope, was a user fluke, is gone now.